packages/data-stores/src/plans/types.ts

```diff
@@ -162,6 +164,15 @@ export interface PricedAPIPlanIntroductoryOffer {
 	introductory_offer_end_date?: string;
 }
 
+export interface CostOverrides {
```

Comment:
Can we rename this to `PricedAPIPlanCostOverrides`? The suffixes are a bit of a pattern in this - to distinguish between API-fetched data and frontend ones.